browse/templates/abs/abs.html

```diff
@@ -171,6 +172,16 @@ <h2>Submission history</h2> From: {{ abs_meta.submitter.name|tex_to_utf if abs_m
 {%- endif -%}
 {%- endmacro -%}
 
+{%- macro generate_social_media_tags() -%}
+    <meta name="twitter:site" content="@arxiv"/>
+    <meta property="twitter:title" content="{{ abs_meta.title|tex_to_utf|truncate(70, False, '...') }}"/>
+    <meta property="twitter:description" content="{{ abs_meta.abstract|trim|truncate(200, False, '...')|tex_to_utf }}"/>
+    <meta property="og:site_name" content="arXiv.org"/>
+    <meta property="og:title" content="{{ abs_meta.title|tex_to_utf }}"/>
+    <meta property="og:url" content="https://arxiv.org/{{ requested_id }}"/>
```

Comment:
Is this right? Should this use ``url_for()`` so that the base server is take into account?

Reply:
By "is this right" I mean "what is requested_id and will this generate a usable url"

Reply:
How about `<meta property="og:url" content="{{ abs_meta.arxiv_identifier.canonical_url() }}"/>` ?